src/core/positioning/positioning.cpp

```diff
@@ -191,20 +192,22 @@ void Positioning::setupDevice()
   }
   else
   {
+    const qsizetype portSeparator = mDeviceId.lastIndexOf( ':' );
+    const QString address = mDeviceId.mid( 4, portSeparator - 4 );
+    const int port = mDeviceId.mid( portSeparator + 1 ).toInt();
```

Comment:
As discussed in chat, we don't want a port / address configuration for egeniouss. You'll have to move that back into tcp / udp conditional blocks.